spec/features/response_index_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+feature "response index", js: true do
+  include_context "search"
+  include_context "batch delete"
+  let(:admin) { create(:admin) }
+  let(:mission) { get_mission }
+
+  before do
+    login(admin)
+  end
+
+  describe "bulk destroy not paginated" do
+    let!(:responses) { create_list(:response, 5, mission: mission) }
+
+    context "unfiltered" do
+      it_behaves_like "select all on page", link: "Delete Selected", klass: "responses", num: 5
+    end
+
+    context "filtered" do
```

Comment:
I don't think you need these context blocks if there is only an `it_behaves_like` inside. And you can I think move the before block up one level here without breaking the other specs, right?